README.md

```diff
@@ -1,13 +1,85 @@
+FireCloud method: [vanallenlab/facets](https://portal.firecloud.org/#methods/vanallenlab/facets/)
+
+## Task details
+
+### Pileup
+Read depth and allele counts at sites of common single nucleotide variant are observed in both the tumor and normal bams based on a provided VCF. 
+
+Inputs:
+- Pair name
+- Normal bam and corresponding index
+- Tumor bam and corresponding index
+- VCF and corresponding index for common variants
+- Minimum mapping quality
+- Minimum based quality
+- Minimum read depth in normal and tumor
+- Number of pseudo SNPs
+
+Read the [pileup documentation](https://github.com/mskcc/facets/tree/master/inst/extcode) for more details of each input. The default VCF of common variants is [human 9606 b151 GRCh37](ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b151_GRCh37p13/VCF/), if you do not have access please download the appropriate VCF and host in a bucket that you have access to.
+
+Outputs:
+- Pileup in gunzip format. Read more in the [userguide](https://github.com/mskcc/facets/blob/master/vignettes/FACETS.pdf).
+
+[Relevant codeblock in WDL](https://github.com/vanallenlab/facets/blob/lab_harmonize/facets.wdl#L95-L138)
+
+### FACETS
+Estimates fraction and allele specific copy number from paired tumor/normal sequencing. As a result, also estimates purity and ploidy of a given tumor sample. This implementation will run FACETS 10 times across different seeds to observe how stable the inferred purity and ploidy values are for a given pair, the seed with a purity closest to the median purity value is selected. 
+
+Inputs:
+- Pair name
+- Pileup
+- Minimum read depth in normal
+- Critical value (cval), the threshold for determining if a change exists
+- Maximum number of expectation maximization iterations (maxiter)
+- Initial seed to set R to
+- Number of seeds to test
+
+Outputs:
+- Genome segments plot, displaying copy and log ratios across chromosomes
+- Diagnostics plot, displaying the segment summaries
+- Iterations plot, displys ploidy and purity for all iterations performed across seeds
+- Copy number cellular fraction, total and minor allele counts for all segments
+- Estimated purity and ploidy
+- Flags and warnings generated by FACETS
+- Seed value closest to median purity
+- Number of seeds that resulted in purity values equaling NA
+
+[Relevant codeblock in WDL](https://github.com/vanallenlab/facets/blob/lab_harmonize/facets.wdl#L140-L180)
+
+### Infer Whole-genome doubling
+Infers whole genome doubling based on [Bielski CM, Zehir A, Penson AV, et al. Genome doubling shapes the evolution and prognosis of advanced cancers](https://doi.org/10.1038/s41588-018-0165-1). Calculates major copy number (MCN) estimate based on total copy number (TCN) estimate and minor copy number (LCN) estimate from FACETS and calls whole-genome doubling if the average MCN across the genome is greater than 2. In cases that LCN is equal to NA, a value of 1 is used to be conservative in the calculation of MCN.
```

Comment:
@brendanreardon @JakeConway I definitely don't understand this issue, but should we not exclude LCA = NA sites? Is there danger that the true LCN in these cases is > 1 (and so MCN might be overestimated)?

Reply:
I don't think that we should be excluding LCN = NA sites because we can infer enough based on TCN alone for the scope of this task, plus we need to normalize all MCN values by their segment lengths to get the fraction of MCN > 2 across the genome. I think that the only blind spot is when the true TCN = 4 and LCN = 2, but that should indistinguishable from a diploid state. Anyway, I'll e-mail Craig to double check none the less.